concepticondata/conceptlists/Gruehn-2008-200.tsv

```diff
@@ -0,0 +1,201 @@
+Gruehn-2008-200-89	89	gesund	healthy	1364	HEALTHY	15.579	6.917	6.083	6.75	6.333	6.5	6.542	6.521	2.167	2	2.167	3	2.083	2.583	2.333	5.5	5.917	4.083	4.5	5.708	4.292	5	6	4.917	4.667	5.25	5.458	4.958	5.208	5.917	5.917	4.917	4.5	5.917	4.708	5.313	1.5	1.667	1.917	2.167	1.583	2.042	1.813	6.875	6	2
+Gruehn-2008-200-90	90	gewissenhaft	conscientious			1.924	5.333	4.833	6.5	6.25	5.083	6.375	5.729	4.583	4.75	3.667	4.333	4.667	4	4.333	5.583	5.583	5.5	6	5.583	5.739	5.66	2.917	2.833	4.167	3.583	2.875	3.875	3.375	6	4.667	6.167	5.833	5.333	6	5.667	3.583	3.833	3.833	4	3.708	3.917	3.813	5.938	5.938	1.75
+Gruehn-2008-200-91	91	gierig	greedy	2018	GREEDY	1.732	2.167	1.917	1.583	1.583	2.042	1.583	1.813	6.083	6.25	6.083	6.167	6.167	6.125	6.146	2.833	2.909	4.583	3	2.87	3.792	3.34	4.917	6	4.417	5.167	5.458	4.792	5.125	2.583	3.333	1.417	1.583	2.958	1.5	2.229	2.417	2.083	2	2.333	2.25	2.167	2.208	2.25	1.813	1.25
+Gruehn-2008-200-92	92	gluecklich	happy	1495	HAPPY	40.126	6.917	6.75	6.917	6.583	6.833	6.75	6.792	2.333	1.667	1.417	2.167	2	1.792	1.896	5.25	4.667	3.667	4.583	4.958	4.125	4.542	6.167	5.583	6.364	5.364	5.875	5.864	5.87	6	5.667	5.25	5.75	5.833	5.5	5.667	2.917	3	2.833	3.25	2.958	3.042	3	6.938	6.688	2
+Gruehn-2008-200-93	93	grausam	cruel	500	CRUEL	5.978	1	1.167	1	1.25	1.083	1.125	1.104	6.333	6.5	6.833	6.583	6.417	6.708	6.563	3.833	4.25	4.667	3.5	4.042	4.083	4.063	5.75	6.167	5.583	6	5.958	5.792	5.875	1	2	1.091	1.417	1.5	1.261	1.383	2.917	2.75	2.75	2.75	2.833	2.75	2.792	1.313	1.063	1.083
+Gruehn-2008-200-94	94	gruendlich	thorough			15.635	4.75	4.667	6.417	6.083	4.708	6.25	5.479	4.417	4.75	3.917	3.583	4.583	3.75	4.167	5.583	5.667	5.333	5.333	5.625	5.333	5.479	3.5	4.417	4.417	3.667	3.958	4.042	4	5.417	4.583	6.083	6	5	6.042	5.521	3.667	3.5	3.333	3.917	3.583	3.625	3.604	5.813	5.375	1.917
+Gruehn-2008-200-95	95	guetig	kind	3299	KIND	1.511	6.25	5.75	6.833	6.083	6	6.458	6.229	1.917	2.167	2.25	2.273	2.042	2.261	2.149	5.25	5.5	4.833	4	5.375	4.417	4.896	3.833	3.583	5.25	4.182	3.708	4.739	4.213	5.083	4.583	5.667	5.083	4.833	5.375	5.104	4.083	3.833	3.833	4.091	3.958	3.957	3.957	5.813	5.938	1.083
+Gruehn-2008-200-96	96	harmlos	harmless			7.908	4.417	4.417	4.667	4.333	4.417	4.5	4.458	1.917	1.667	3	2.167	1.792	2.583	2.188	3.75	3.333	3.417	2.75	3.542	3.083	3.313	3.333	3.583	4	2.273	3.458	3.174	3.319	4.75	4.083	4	3.083	4.417	3.542	3.979	3.5	3.75	3.333	3.833	3.625	3.583	3.604	4.125	4.5	1.167
+Gruehn-2008-200-97	97	haesslich	ugly	1548	UGLY	3.174	1.75	2	2.333	2.25	1.875	2.292	2.083	4.25	4.1	4.417	4.5	4.625	4.458	4.542	3.25	2.75	2.917	3	3	2.957	2.979	6.333	6.75	5.417	6.833	6.542	6.125	6.333	2.083	1.75	2.417	2.833	1.917	2.625	2.271	3.25	3.167	3.083	3.667	3.208	3.375	3.292	2	3.188	1.708
+Gruehn-2008-200-98	98	heiter	bright			9.268	6.083	5.917	6.167	6.083	6	6.125	6.063	2.667	2.833	3	3.182	2.75	3.087	2.915	5.167	4.417	4.167	4.25	4.792	4.208	4.5	5.583	5.583	5.917	5.333	5.583	5.625	5.604	6	5.583	5.417	5.167	5.792	5.292	5.542	2.917	2.75	2.833	2.667	2.833	2.75	2.792	6.125	5.875	2
```

Comment:
Our mappings are not consistent here, we've got two cases of _heiter_, one mapped to `HAPPY`, the other to `MERRY`.

Reply:
I interpreted the word "heiter" here as "heiteres Wetter" due to the English translation "bright".

Reply:
Ah, I interpreted all of them as pertaining to qualities (etc.) of people, bright as in "cheerful and lively".